SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -215,6 +215,10 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 	inline bool IsAuthoritativeServer() const { return bIsAuthServer; }
 
+	FORCEINLINE bool IsAutonomousProxyOnAuthority() const { return bIsAutonomousProxyOnAuthority; }
```

Comment:
nit: Generally we don't use forceinline until a profiled run has shown it's necessary, so I'd removed these